src/main/java/dev/dokan/dokan_java/masking/MaskValueSet.java

```diff
@@ -51,4 +73,8 @@
 
     void add(T... items);
 
+    EnumSet<T> elements();
```

Comment:
I don't understand for what use case this method is defined in the interface.

Reply:
Mostly internal use. Also I was under the impression that a converter method could be useful.

Reply:
As long as its only for internal usage and someone doesn't have a need for it, i suggest to _not_ add the method to the interface.

Reply:
Usually internal usage should aim to use the interfaces, too.
As I said: Converters can be useful. Giving a user the ability to use EnumSet can't be really wrong. The JDK has over 300 references to it.

Reply:
I assume this isn't a dealbreaker. I'm pretty sure it's better to have the method than not to have it. We can remove it later if we want.